charts/cloudzero-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: cloudzero-agent
 description: A chart for using Prometheus in agent mode to send cluster metrics to the CloudZero platform.
 type: application
-version: 0.0.0-dev
+version: 1.0.0
```

Comment:
do we want to keep this as is given our conversation about the goal of this version?

Reply:
I'll revert this, thanks for the catch!